src/libstore/uds-remote-store.hh

```diff
@@ -9,16 +9,43 @@ namespace nix {
 
 struct UDSRemoteStoreConfig : virtual LocalFSStoreConfig, virtual RemoteStoreConfig
 {
-    UDSRemoteStoreConfig(const Params & params)
+
+    protected:
+        static constexpr char const * scheme = "unix";
+    
+    public:
```

Comment:
```suggestion
protected:
    static constexpr char const * scheme = "unix";
    
public:
```
it's a bit weird looking, but that's how this is generally formatted (pre and post the formatter).